### PR TITLE
Galex data available in S3

### PR DESCRIPTION
**Pull request by sshen-stsci**
*Issue #, if available:*

*Description of changes:*

Add Galex dataset with related description and license text.
 
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jflasher** [comment]
Hi @sshen-stsci, can you mark this post as a Draft and then take it out of that status when you're ready to have it reviewed? Thanks!

**Comment by sshen-stsci** [comment]
Ready to merge the changes in branch gales-patch-1 into main branch

File: datasets/galex.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a ['GALEX'](https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html) bucket
+    ARN: arn:aws:s3:::stpubdata/galex
+    Region: us-east-1
+    Type: S3 Bucket
+    RequesterPays: True
```
Comment (jflasher) [inline datasets/galex.yaml:17]: You can remove this line as it is not in a Requester Pays bucket.

File: datasets/galex.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+UpdateFrequency: Never
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a ['GALEX'](https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html) bucket
```
Comment (jflasher) [inline datasets/galex.yaml:13]: You have a link to Requester Pays here but you can drop that. You're likely fine just having a description like `GALEX project data files.`

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @sshen-stsci, thanks for submitting. I had two comments inline and then you will also need to pull in the latest from the `main` branch, and then this should be good to merge!

**Comment by sshen-stsci** [comment]
Suggested changes have been made and merged to the branch 'gale-patch-1"

File: datasets/galex.yaml (hunk 1 above)
Comment (sshen-stsci) [reply datasets/galex.yaml]: updated

**Comment by jflasher** [comment]
Thanks @sshen-stsci, merging, will be live in a few minutes!

**Pull request merged**